distribution/docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM maven:3-jdk-8 as builder
+
+COPY . /src
+WORKDIR /src
+RUN mvn install -ff -DskipTests -Dforbiddenapis.skip=true -Pdist -Pbundle-contrib-exts
+RUN \
+ VER=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive exec:exec) \
+ && tar -zxf ./distribution/target/apache-druid-${VER}-bin.tar.gz -C /opt \
+ && ln -s /opt/apache-druid-${VER} /opt/druid
+
+RUN cd /opt/druid/extensions/mysql-metadata-storage \
```

Comment:
Can there be no `cd` command? it would be nice if the target directories could be specified in the commands themselves. Like `wget -O /some/dir/something.jar`